libsolidity/ast/Types.cpp

```diff
@@ -2683,6 +2684,7 @@ unsigned FunctionType::sizeOnStack() const
 	case Kind::BareDelegateCall:
 	case Kind::Internal:
 	case Kind::ArrayPush:
+	case Kind::ArrayPop:
```

Comment:
This change the size of the function on the stack which is a rather serious thing. I'll add another test.